packages/market-service/src/foxy/foxy.ts

```diff
@@ -52,8 +51,11 @@ export class FoxyMarketService implements MarketService {
         return null
       }
 
-      const { data } = await axios.get(`${this.baseUrl}/assets/${FOX_COINCAP_ID}`)
-      const marketData = data.data as CoinCapMarketCap
+      const cgData = await super.findByAssetId({
```

Comment:
```suggestion
      const coinGeckoData = await super.findByAssetId({
```

Reply:
Renamed in bb6126bafa3f6fcb6e0ca3de88b08930d0ead244